sdk/sdk-core/aar/src/main/java/mil/emp3/core/services/kml/KMZFile.java

```diff
@@ -53,6 +55,15 @@ protected void unzipKMZFile(IKMZFileRequest request) throws EMP_Exception {
                     File directory = new File(request.getDestinationDir() + File.separator + zipEntry.getName());
                     directory.mkdirs();
                     continue;
+                 //It is possible for the zip to not contain the directory but files under the
+                 //directory.  This case generates the directory in that case
+                } else if(zipEntry.getName().contains(File.separator)) {
+                    File file_in_zip = new File(zipEntry.getName());
```

Comment:
keep variable naming conventions the same.  rename file_in_zip to fileInZip (since we are using camel casing everywhere else).

Reply:
also add final before all these variables